src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts

```diff
@@ -91,26 +88,46 @@ export function useCreateSarcophagus() {
     uploadFile,
   ]);
 
+  const handleCreate = useCallback(async () => {
```

Comment:
I'm assuming this step is separate because you have to connect to a different network first, like we were talking about in stand up.

Reply:
Correct